src/Expensify.js

```diff
@@ -34,6 +34,7 @@ Onyx.init({
         [ONYXKEYS.SESSION]: {loading: false},
         [ONYXKEYS.ACCOUNT]: {loading: false, error: ''},
         [ONYXKEYS.NETWORK]: {isOffline: false},
+        [ONYXKEYS.IOU]: {isLoading: true},
```

Comment:
Why is the default `true` and not `false` - since we set it to true when the network request starts?

Reply:
Agree that this should be false

Reply:
Fixed 👍